Mathlib/Geometry/Manifold/ContMDiff/Defs.lean

```diff
@@ -25,7 +25,7 @@ Let `M` and `M'` be two smooth manifolds, with respect to model with corners `I`
 
 We also give some basic properties of smooth functions between manifolds, following the API of
 smooth functions between vector spaces.
-See `Basic.lean` for further basic properties of smooth functions between smooth manifolds,
+See `Basic.lean` for further basic properties of smooth functions between manifolds,
```

Comment:
I'm thinking: should you say 
```
We also give some basic properties of C^n functions between manifolds, following the API of
C^n functions between vector spaces.
See `Basic.lean` for further basic properties of C^n functions between manifolds,
```
instead? (Perhaps the "smoothness" in the lines following this can stay, but these ones feel a bit out of place.)

Reply:
Line 35, "smooth groupoid": please change that for sure